src/views/ConsentContinuationOut.vue

```diff
@@ -575,6 +610,13 @@ export default class ConsentContinuationOut extends Mixins(CommonMixin, DateMixi
         // Show error message of legal name text field if invalid
         this.$refs.certifyRef.$refs.certifyTextfieldRef.error = true
       }
+      if (!this.foreignJurisdictionValid) {
+        // Show error message of legal name text field if invalid
+        this.$refs.foreignJurisdictionRef.$refs.countrySelectRef.error = true
+        if (this.selectedCountry) {
+          this.$refs.foreignJurisdictionRef.$refs.regionSelectRef.error = true
+        }
+      }
```

Comment:
Validations:

![image](https://github.com/bcgov/business-filings-ui/assets/116035339/3bea5de1-522a-42bb-be30-ca6fc40a4325)

![image](https://github.com/bcgov/business-filings-ui/assets/116035339/890d0982-c276-4aeb-bc0a-4e390510cbcf)

Reply:
I think the sub-component should validate itself. It should not be this component's responsibility.

Reply:
Sorry to jump in on here. I think this is only to show the error once the user presses on `File Now`. This is a UX thing Yui mentioned to me in a previous ticket. This essentially is just to show the error.

Reply:
Without this, error messages do not show until we select the box.

Reply:
This code is reaching way into the sub-component to explicitly set the error flag. This component should not need to know so much about what valid means or what sub-sub-components are under it.

Other sub-components take a prop, which they watch, to validate their values.

Reply:
This is now resolved. (Thanks!)